packages/lesswrong/components/posts/PostsItemMeta.tsx

```diff
@@ -51,6 +54,11 @@ const PostsItemMeta = ({post, read, classes}: {
       </span>}
 
       { post.isEvent && <span className={classes.info}>
+        {post.startTime && post.endTime && (
```

Comment:
This should probably be only checking for startTime, not for both? I know some other parts of the code treat the end-time as optional, so events with a start time but no end time do exist. Here's an example: https://www.lesswrong.com/posts/RsTyedqDCWf3tp8Q9 . For a Google Calendar event, in that case, you want a default duration of 1hr or something.

Reply:
Yeah, on reflection, I think a default of 1hr for calendar event is maybe better than enforcing having an end-time.